data_collections/csv_updater.py

```
@@ -1,8 +1,38 @@
+def extract_entries_from_csv(path: str):
+    """
+    Extract entries from `runningCSV.csv` and return a list of dict.
+    """
+    entries_from_csv = []
+    with open(path) as file:
+        csv_file = csv.DictReader(file)
+        for row in csv_file:
+            entries_from_csv.append(row)
+    return entries_from_csv
+
+def removeDuplicates(data: list[dict]):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)